README.md

```diff
@@ -178,6 +178,26 @@ The "Delete Staging Deployment" action allows you to delete the deployment not r
           app-name: <app name>
 ```
 
+## Arguments
+> [!NOTE]
+> Some arguments are only applicable for certain settings of the `action` argument. The Action column below specifies the pertinent actions for each argument. Any argument listed as Required is only required for the pertinent Action(s).
+
+|Argument|Action&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|Description|
```

Comment:
Remove `&nbsp;` s

Reply:
Removed, but the Action column will wrap at set-production